frontend/www/js/omegaup/course/mine.ts

```diff
@@ -0,0 +1,26 @@
+import course_List from '../components/course/Mine.vue';
+import { omegaup, OmegaUp } from '../omegaup';
+import { types } from '../api_types';
+import * as api from '../api';
+import * as ui from '../ui';
+import T from '../lang';
+import Vue from 'vue';
+
+OmegaUp.on('ready', () => {
+  const payload = types.payloadParsers.CourseListMinePayload();
+  const headerPayload = types.payloadParsers.CommonPayload();
+  let courseList = new Vue({
+    el: '#main-container',
+    render: function (createElement) {
+      return createElement('omegaup-course-list', {
+        props: {
+          courses: payload.courses,
+          isMainUserIdentity: headerPayload?.isMainUserIdentity,
```

Comment:
como esto está marcado como booleano, hay que asegurarnos que sea un booleano en vez de `boolean | undefined`. El operador `?.` regresa `undefined` si el miembro no se encuentra presente: https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#optional-chaining

```suggestion
          isMainUserIdentity: headerPayload?.isMainUserIdentity || false,
```